lightning/src/routing/scorer.rs

```diff
@@ -98,16 +98,37 @@ pub struct ScorerUsingTime<T: Time> {
 /// Parameters for configuring [`Scorer`].
 pub struct ScoringParameters {
 	/// A fixed penalty in msats to apply to each channel.
+	///
+	/// Default value: 500 msat
 	pub base_penalty_msat: u64,
 
 	/// A penalty in msats to apply to a channel upon failing to relay a payment.
 	///
 	/// This accumulates for each failure but may be reduced over time based on
 	/// [`failure_penalty_half_life`].
 	///
+	/// Default value: 1,024,000 msat
+	///
 	/// [`failure_penalty_half_life`]: Self::failure_penalty_half_life
 	pub failure_penalty_msat: u64,
 
+	/// When the amount being sent over a channel is this many 1024ths of the total channel value,
+	/// we begin applying [`overuse_penalty_msat_per_1024th`].
+	///
+	/// Default value: 128 1024ths (i.e. begin penalizing when an HTLC uses 1/8th of a channel)
+	///
+	/// [`overuse_penalty_msat_per_1024th`]: Self::overuse_penalty_msat_per_1024th
+	pub overuse_penalty_start_1024th: u16,
+
+	/// A penalty applied, per whole 1024ths of the channel value which the amount being sent over
+	/// the channel exceeds [`overuse_penalty_start_1024th`] by.
+	///
+	/// Default value: 20 msat (i.e. 2560 msat penalty to use 1/4th of a channel, 7680 msat penalty
+	///                to use half a channel, and 12,560 msat penalty to use 3/4ths of a channel)
+	///
+	/// [`overuse_penalty_start_1024th`]: Self::overuse_penalty_start_1024th
+	pub overuse_penalty_msat_per_1024th: u64,
```

Comment:
Related to earlier comment, here we are saying "channel value". Should be consistent with how it is defined in `Score`.